libraries/botbuilder/tests/teamsHelpers.test.js

```diff
@@ -195,4 +196,35 @@ describe('TeamsActivityHelpers method', function () {
             assert.strictEqual(channelId, '');
         });
     });
+
+    describe('teamsGetTeamOnBehalfOf()', function () {
+        it('should return onBehalfOf list', async function () {
+            const activity = {
+                channelData: {
+                    onBehalfOf: [
+                        {
+                            itemId: 0,
+                            displayName: 'onBehalfOfTest',
+                            mentionType: 'person',
+                            mri: 'mriTest',
+                        },
+                    ],
+                },
+            };
+            const onBehalfOf = teamsGetTeamOnBehalfOf(activity)[0];
+            assert.strictEqual(onBehalfOf.displayName, 'onBehalfOfTest');
+        });
+
+        it('should return null with no channelData', async function () {
+            const activity = createActivityNoChannelData();
+            const onBehalfOf = teamsGetTeamOnBehalfOf(activity);
+            assert(onBehalfOf === null);
+        });
+
+        it('should return null with no onBehalfOf list', async function () {
+            const activity = { channelData: { onBehalfOf: null } };
+            const onBehalfOf = teamsGetTeamOnBehalfOf(activity);
+            assert.strictEqual(onBehalfOf, null);
+        });
```

Comment:
These 2 tests are checking for null in 2 different ways. Which one is used in the rest of the tests? Let's keep it consistent.